src/_components/TmrForm/index.tsx

```diff
@@ -29,8 +29,14 @@ const TmrForm = (props: TmrFormProps) => {
       <h1 className="text-xl font-bold mb-6">{heading}</h1>
       <form onSubmit={handleSubmit(onSubmit)}>
         {inputs.map((input, index) => {
-          const { label, type, placeholder, errorOnEmpty, validateError } =
-            input;
+          const {
+            label,
+            type,
+            placeholder,
+            errorOnEmpty,
+            helperText,
+            validateError,
+          } = input; // Destructure the input object
```

Comment:
Mình k cần cái comment ở đây nha. Nên xoá cái comment này di
`// Destructure the input object`